platform/jvm/microbenchmark/src/androidTest/java/io/bitdrift/microbenchmark/LogBenchmarkTest.kt

```diff
@@ -39,46 +36,27 @@ private const val LOG_MESSAGE = "50 characters long test message - 0123456789012
  * In microbenchmarks the number of iterations to be executed is determined by the library itself
  */
 @RunWith(AndroidJUnit4::class)
-class ClockTimeProfiler {
+class LogBenchmarkTest {
     @get:Rule
     val benchmarkRule = BenchmarkRule()
 
-    private fun startLogger() {
+    private fun startLogger(fieldProviders: List<FieldProvider> = listOf()) {
+        CaptureJniLibrary.load()
         Capture.Logger.start(
-            apiKey = "android-benchmark-test",
-            apiUrl = "https://api-tests.bitdrift.io".toHttpUrl(),
+            apiKey = "[test_api_key]",
+            apiUrl = "https://api-test.bitdrift.dev".toHttpUrl(),
+            context = InstrumentationRegistry.getInstrumentation().targetContext,
             sessionStrategy = SessionStrategy.Fixed(),
+            fieldProviders = fieldProviders
         )
     }
 
-    @Test
-    fun loggerStart() {
```

Comment:
deletes this test altogether since it wasn't really proving valuable (the JNI library loads only once anyway)